### PR TITLE
Activate nmstate by default

### PR DESCRIPTION
**Pull request by qinqon**
Also upgrade nmstate contianer to 0.6.0

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: pkg/controller/hyperconverged/factory.go [hunk 1]
```diff
@@ -88,6 +88,7 @@ func newNetworkAddonsForCR(cr *hcov1alpha1.HyperConverged, namespace string) *ne
 			Multus:      &networkaddons.Multus{},
 			LinuxBridge: &networkaddons.LinuxBridge{},
 			KubeMacPool: &networkaddons.KubeMacPool{},
+			KubeMacPool: &networkaddons.Nmstate{},
```
Comment (djzager) [inline pkg/controller/hyperconverged/factory.go]:
```suggestion
			NMState: &networkaddons.NMState{},
```

File: deploy/operator.yaml [hunk 2]
```diff
@@ -191,7 +191,7 @@ spec:
         - name: SRIOV_CNI_IMAGE
           value: quay.io/kubevirt/cluster-network-addon-sriov-cni:v1.1.0-1.git9e4c973
         - name: NMSTATE_HANDLER_IMAGE
-          value: quay.io/nmstate/kubernetes-nmstate-handler:v0.3.0
+          value: quay.io/nmstate/kubernetes-nmstate-handler:v0.6.0
```
Comment (djzager) [inline deploy/operator.yaml]: Unfortunately, it's the `hack/build-manifests.sh` script in conjunction with `manifest-templator.go` that build these files. If you want this change to survive the next time someone builds manifests, you will need to:

1. update it in cluster-network-addons-operator [here](https://github.com/kubevirt/cluster-network-addons-operator/blob/21f2cbaef0e6ef8c292c7c70070fc86d9b6e92db/pkg/components/components.go#L28)
1. bump vendored version of cluster-network-addons-operator in this project
1. rebuild manifests with `hack/build-manifests.sh`

Comment (qinqon) [reply deploy/operator.yaml]: I have the cluster-network-addons-operator here https://github.com/kubevirt/cluster-network-addons-operator/pull/202

Comment (rthallisey) [reply deploy/operator.yaml]: @qinqon looks like the PR merged.  When the CNA creates a new release we can pick it up.

Comment (qinqon) [reply deploy/operator.yaml]: @rthallisey We need also this https://github.com/kubevirt/cluster-network-addons-operator/pull/206 to have new releas of addons operator

**Comment by phoracek** [comment]
@djzager while this PR is still WIP since we need to fix some bits on the operator, we really want to make it to the next release of HCO. Could you guys please ping me before you release, so we can try to get as much as possible in there. Thanks.

**Comment by djzager** [comment]
> @djzager while this PR is still WIP since we need to fix some bits on the operator, we really want to make it to the next release of HCO. Could you guys please ping me before you release, so we can try to get as much as possible in there. Thanks.

seems reasonable. Adding @rthallisey for visibility.

**Comment by phoracek** [comment]
Can be closed in favour of https://github.com/kubevirt/hyperconverged-cluster-operator/pull/214

**Comment by djzager** [comment]
Closing in favor of #214

**Pull request closed without merging**